docker/Dockerfile-pgbase

```diff
@@ -0,0 +1,15 @@
+FROM python:2.7
+RUN apt-get update && apt-get install -y \
+    libffi-dev \
+    libssl-dev \
+    postgresql-client
+ENV PYTHONBUFFERED 1
+ENV DATABASE_URL postgres://postgres:postgres@db:5432/postgres
+ENV FLASK_SECRET_KEY GBhQrZfbN9KGMcyLVImGeS2Wtj3SFCdF
+ENV RABBIT_MANAGEMENT_URL http://pulse:15672/api/
+RUN mkdir /code
+WORKDIR /code
+ADD requirements.txt /code/
+RUN pip install --no-cache-dir -r requirements.txt
+ADD . /code/
+RUN python setup.py develop
```

Comment:
Nit: I think `setup.py develop` is generally discouraged now in favour of `pip install -e .` - though I could be misremembering?